torch/distributed/rpc/api.py

```diff
@@ -1,3 +1,5 @@
+__all__ = ["shutdown", "get_worker_info", "remote", "rpc_sync", "rpc_async", "RRef", "AllGatherStates", "method_factory", "new_method"]
```

Comment:
This was added for compliance for new test: https://github.com/pytorch/pytorch/blob/81722f66306aef05334c27bb2a64137d8fa6493a/test/test_public_bindings.py#L284-L296